lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -642,6 +642,38 @@ struct GrandCentralPass
                  SmallVector<VerbatimXMRbuilder> &xmrElems,
                  SmallVector<InterfaceElemsBuilder> &interfaceBuilder);
 
+  // COPIES OF ABOVE FOR NEW FIRRTL.VIEW
```

Comment:
Nit: can the PR change this shouting comment into something more obvious with a start and end? E.g.,:

```
//===----------------------------------------------------------------------===//
// Intrinsic Handling Functions
//
// Note: these are _intentional_ copies of earlier functions.
//===----------------------------------------------------------------------===//

<awesome copied functions>

//===----------------------------------------------------------------------===//
// End of intrinsic handling functions
//===----------------------------------------------------------------------===//
```

It might benefit to add a block of text for the annotation-handling functions.